### DISCUSSION TITLE
SSC; Command in docs not working

### DISCUSSION THREAD

**Discussion by Debug-pro-dev** [Pryaxis/TShock#2253] [Q&A] [answered]
In docs:
> The SuperAdmin account has all permissions on your server. This includes the permission to ignore Server Side Characters (`tshock.ignore.ssc`).
> If you want to start with a fresh character and still have SuperAdmin privileges, create a new group and grant it the following permissions: `*`, `!tshock.ignore.ssc`. This will enable every permission (`*`), then negate the SSC ignoring permission (`!tshock.ignore.ssc`).
> Example: `/group addperm <group name> * !tshock.ignore.ssc`

But it doesn't work.

I enter this command, adds `*` to the group and nothing else happens!

(`terraria 1.4.1.2`)

**Comment by Kojirremer** [comment]
What commands are you entering? Not just the one for the perm, also the one to create the new group.

**Reply by Debug-pro-dev** [reply]
`/group add owner * !tshock.ignore.ssc`

**Comment by Debug-pro-dev** [comment, marked as answer]
The problem has been resolved.

I got confused about accounts because of which my tests did not work.

I was waiting all the time for a visual confirmation of the right of `!tshock.ignore.ssc` via `/group listperm owner`, but it turned out that it works, only it is not displayed
